clients/spark/build.sbt

```diff
@@ -64,11 +64,11 @@ def generateCoreProject(buildType: BuildType) =
         // version will probably continue to work because the C language API
         // is quite stable.  Take the version documented in DataBricks
         // Runtime 7.6, and note that it changes in 8.3 :-(
-        "org.xerial.snappy" % "snappy-java" % "1.1.7.5",
+        "org.xerial.snappy" % "snappy-java" % "1.1.8.2",
```

Comment:
Did you try using `1.1.8.4`?
It's not that I'm a big expert in snappy-java, It's just that the IDE suggested I change to 1.1.8.4 and it seemed to work...